packages/@react-spectrum/button/package.json

```diff
@@ -52,7 +52,8 @@
   },
   "devDependencies": {
     "@adobe/spectrum-css-temp": "3.0.0-alpha.1",
-    "@react-spectrum/test-utils-internal": "3.0.0-alpha.1"
+    "@react-spectrum/test-utils-internal": "3.0.0-alpha.1",
+    "react-dom": "^16.8.0 || ^17.0.0-rc.1 || ^18.0.0 || ^19.0.0"
```

Comment:
why only react-dom and not react?

Reply:
react is already in the peerDeps, so something else will supply it
nothing is 'technically' supplying react-dom, which the test-utils-internal are requesting, so had to add react-dom here just for dev work